src/main/scala/viper/server/ViperConfig.scala

```diff
@@ -58,23 +61,22 @@ class ViperConfig(args: Seq[String]) extends ScallopConf(args) {
     }
   }
 
-  @deprecated
-  val ideMode: ScallopOption[Boolean] = opt[Boolean]("ideMode",
-    descr = ("Used for VS Code IDE. Report errors in json format, and write"
-      + "errors in the format '<file>,<line>:<col>,<line>:<col>,<message>' to"
-      + "a file (see option ideModeErrorFile)."),
+  val backendSpecificCache: ScallopOption[Boolean] = opt[Boolean]("backendSpecificCache",
+    descr = "Use a separate cache for each backend?",
     default = Some(false),
     noshort = true,
     hidden = false
   )
 
-  val backendSpecificCache: ScallopOption[Boolean] = opt[Boolean]("backendSpecificCache",
-    descr = "Use a separate cache for each backend?",
+  @deprecated
+  val ideMode: ScallopOption[Boolean] = opt[Boolean]("ideMode",
```

Comment:
I suggest removing the deprecated options. we don't need them anymore at this point.